swagger.yaml

```diff
@@ -0,0 +1,2651 @@
+          name: max
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: min
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/ms/_zrevrank/:id/:member':
+    get:
+      description: 'Controller: ms. Action: zrevrank.'
+      parameters:
+        - description: TODO
+          in: path
+          name: id
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: member
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/ms/_zscore/:id/:member':
+    get:
+      description: 'Controller: ms. Action: zscore.'
+      parameters:
+        - description: TODO
+          in: path
+          name: id
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: member
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  /ms/_zunionstore:
+    post:
+      description: 'Controller: ms. Action: zunionstore.'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/profiles/:id':
+    put:
+      description: 'Controller: security. Action: createOrReplaceProfile.'
+      parameters:
+        - description: TODO
+          in: path
+          name: id
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/profiles/:id/_createOrReplace':
+    put:
+      description: 'Controller: security. Action: createOrReplaceProfile.'
+      parameters:
+        - description: TODO
+          in: path
+          name: id
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  /profiles/_create:
+    post:
+      description: 'Controller: security. Action: createProfile.'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  /profiles/_mget:
+    post:
+      description: 'Controller: security. Action: mGetProfiles.'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  /profiles/_search:
+    post:
+      description: 'Controller: security. Action: searchProfiles.'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/roles/:id':
```

Comment:
missing also methods for route '/roles/:id':
*  "get" => security/getRole
* "post" => security/updateRole
* "delete" => security/deleteRole